### PR TITLE
BUG: Fixed biplot scaling issue

### PR DESCRIPTION
**Pull request by mortonjt**
This fixes this issue https://github.com/biocore/emperor/issues/373

### CODE REVIEW

**Comment by emperor-helper** [comment]
The test build for this pull request can be found here: http://emperor.colorado.edu/pull_386/make_emperor/

File: emperor/support_files/js/emperor.js [hunk 1]
```diff
@@ -183,6 +183,23 @@ function toggleScaleCoordinates(element) {
...
+    g_mainScene.remove(g_plotTaxaArrows[index]);
+
+    var whiteColor = new THREE.Color();
+    whiteColor.setHex("0xFFFFFF");
+    var taxaVector = makeLine([0, 0, 0,],
+			      [g_plotTaxa[index].position.x,
+			       g_plotTaxa[index].position.y,
+			       g_plotTaxa[index].position.z],
+			      whiteColor, 2);
+
```
Comment (ElDeveloper) [inline emperor/support_files/js/emperor.js]: Can you remove this extra whitespace?

File: emperor/support_files/js/emperor.js [hunk 2]
```diff
@@ -183,6 +183,23 @@ function toggleScaleCoordinates(element) {
       operation(g_plotTaxa[index].scale.x, g_fractionExplained[g_viewingAxes[0]]),
       operation(g_plotTaxa[index].scale.y, g_fractionExplained[g_viewingAxes[0]]),
       operation(g_plotTaxa[index].scale.z, g_fractionExplained[g_viewingAxes[0]]));
+
+    //scale the dimensions of the taxa vectors
+    g_elementsGroup.remove(g_plotTaxaArrows[index])
```
Comment (ElDeveloper) [inline emperor/support_files/js/emperor.js]: Can you add a semicolon at the end of this line?

File: emperor/support_files/js/emperor.js [hunk 3]
```diff
@@ -183,6 +183,23 @@ function toggleScaleCoordinates(element) {
...
+    whiteColor.setHex("0xFFFFFF");
+    var taxaVector = makeLine([0, 0, 0,],
+			      [g_plotTaxa[index].position.x,
+			       g_plotTaxa[index].position.y,
+			       g_plotTaxa[index].position.z],
+			      whiteColor, 2);
+
+
+    g_plotTaxaArrows[index] = taxaVector;
+    g_elementsGroup.add(g_plotTaxaArrows[index])
```
Comment (ElDeveloper) [inline emperor/support_files/js/emperor.js]: Can you add a semicolon at the end of this line?

**Comment by ElDeveloper** [comment]
Looks good @mortonjt, tested the interface and it seems to be working great! Do you think it would make sense to add a note to the ChangeLog.md file?

File: emperor/support_files/js/emperor.js (hunk 1 above)
Comment (mortonjt) [inline emperor/support_files/js/emperor.js]: That's weird.  It shows up differently in my editor ...

**Comment by ElDeveloper** [comment]
Just FYI, @antgonza mad a comment here: https://github.com/biocore/qiita/issues/1023

**Comment by mortonjt** [comment]
I think it would make sense to add this to the Changelog.  Thanks!

File: emperor/support_files/js/emperor.js (hunk 1 above)
Comment (ElDeveloper) [inline emperor/support_files/js/emperor.js]: Hmmm, I just pulled down your code into my machine and it shows up exactly the same way as it's showing on GitHub:

screen shot 2015-03-31 at 2 45 39 pm

File: ChangeLog.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ Emperor 0.9.51-dev (changes since Emperor 0.9.5 go here)
 --------------------------------------------------------
 
 ### Bug Fixes
-
+* Fix problem where the taxonomic vectors weren't scaled properly when the scale coordinates button was toggled
```
Comment (ElDeveloper) [inline ChangeLog.md]: Thanks, can you add a period at the end of the sentence and reference the issue number?

**Comment by ElDeveloper** [comment]
I added two minor comments, @antgonza would you mind merging when it's ready? you encountered an issue that I didn't so it would be great to verify this is no longer present.

**Pull request merged**